### PR TITLE
Added PWA and updated for Native Mobile

### PR DESCRIPTION
**Pull request by dannyroestmx**

### CODE REVIEW

File: content/evaluation-guide/app-capabilities/progressive-web-apps.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Mendix provides several common patterns and best practices per device to help you build great user experiences. You can also extend these patterns with custom variants, as discussed in [User Interface Design](ui-design).
+
+## 3 How Can I Extend My Progressive Web App?
+
+Mendix offers several options for extending web apps. These are discussed in the section [How Can I Extend the Mendix Front-End?](front-end#extend) of *Front-End* and the section [How Can I Customize the Look and Feel of My Apps?](ui-design#customize) of *User Interface Design*.
+
+Based on settings a default [Web App Manifest](https://www.w3.org/TR/appmanifest/) is generated, but developer can fully customize if needed. Next to that, both the *index.html* and login page can be fully customized to your needs. 
+
+## 4 How Can I Test or Distribute My Progressive Web App?
```
Comment (ConnorLand) [inline content/evaluation-guide/app-capabilities/progressive-web-apps.md:41]: This heading is very similar to the one below it. Is it possible for you to make them a little more distinct @dannyroestmx ?

File: content/evaluation-guide/app-capabilities/native-mobile-apps.md [hunk 2]
```diff
@@ -48,4 +48,22 @@ For more information, see [How to Get Started with Native Mobile](https://docs.m
...
+
+## 5 How Can I Build and Distribute My Native Mobile App {#how-can-build-my-mobile-app}
+
+Mendix offers a fully UI driven approach to build the binaries for your native mobile app. Directly from Studio pro a developer can launch the UI to build a binary for development or for distrubtion. Using a simple wizard or through advanced settings developer can easily configure several settings like, app name and id, icons, splash screens, push notifications, api keys, permissions and more. Developers can also create different profiles to fascilitate for different environments or other variants.
+
+{{% image_container width="650" %}}
+![User interface for mobile app building](attachments/native_app_building_wizard.png)
+{{% /image_container %}}
+
+Via this user interface, the app can be build automatically via [Microsoft Visual Studio App Center](https://appcenter.ms/). Via this approach the developer does not need
```
Comment (ConnorLand) [inline content/evaluation-guide/app-capabilities/native-mobile-apps.md]: What was the rest of the thought after "Via this approach the developer does not need" @dannyroestmx ?

File: content/evaluation-guide/app-capabilities/progressive-web-apps.md (hunk 1 above)
Comment (dannythinkyou) [reply content/evaluation-guide/app-capabilities/progressive-web-apps.md:41]: Can be changed to How Can I Distribute My Progressive Web App

File: content/evaluation-guide/app-capabilities/native-mobile-apps.md (hunk 2 above)
Comment (dannythinkyou) [reply content/evaluation-guide/app-capabilities/native-mobile-apps.md]: to install software like XCode or Android Studio. However, if preferred developers can checkout the source and build using the tools of their preference.

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**